tests/vmi_lifecycle_test.go

```diff
@@ -604,7 +604,7 @@ var _ = Describe("[rfe_id:273][crit:high][vendor:cnv-qe@redhat.com][level:compon
 					return nil
 				}, 100, 10).Should(Succeed(), "Should be able to start a new VM")
 
-				By("checking if the VMI with bridged networking is still running, it will verify the CNI didn't cause the pod to be killed")
+				By("checking if the VMI with bridged networking is still running.   Some CNIs will cause pods to be in a failed state")
```

Comment:
@rthallisey according to your explanation it is indeed the CNI moving the pod to `Failed` state and not kubelet. So I would keep the previous version (in the current version it seems there are scenarios the pod will be moved to `Failed` state).